data/ingredients.js

```diff
@@ -1233,3 +1233,5 @@ let ingredientsData = [
     "estimatedCostInCents": 613
   }
 ]
+
+module.exports = { ingredientsData };
```

Comment:
we'll need to fix this export in the next sprint.

Reply:
I think it might be okay?  I put the fancy conditional export at the bottom of `Recipe.js` for the browser. But of course `Recipe.js` relies on `ingredients.js` -- so maybe the fancy export will need to be in not just `Recipe.js` but also in all of its dependencies?